deploy/docker/fs/opt/appsmith/healthcheck.sh

```
@@ -25,14 +29,19 @@ while read -r line
             echo 'ERROR: Mongo is down';
             healthy=false
         fi
-       elif [[ "$process" == "redis" ]]; then
+      elif [[ "$process" == "redis" ]]; then
         if [[ $(redis-cli ping) != 'PONG' ]]; then
             echo 'ERROR: Redis is down';
             healthy=false
         fi
+      elif [[ "$process" == "mcp" ]]; then
+        if [[ $(curl -s --max-time 5 -w "%{http_code}\n" http://localhost:${APPSMITH_MCP_PORT:-8092}/health -o /dev/null) -ne 200 ]]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)